stdlib/dynarray.mli

```diff
@@ -0,0 +1,190 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                            Simon Cruanes                               *)
+(*                                                                        *)
+(*   Copyright 2022 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Growable, mutable array.
+
+
+
+    @since 5.1
+*)
+
+type 'a t
+(** A dynamic array containing values of type ['a].
+
+    This contains an underlying {!array} along with a size.
+    Operations such as {!push}, {!append}, and {!append_seq}, extend the
+    size (and might reallocate the underlying array).
+
+    Operations such as {!pop}, and {!truncate}, reduce the size. *)
+
+val create : unit -> 'a t
+(** Create a new, empty array. *)
```

Comment:
It would be nice if the `[f args…] description` pattern is followed all along. For example, in this particular case `[create ()] is a new empty array`.

Reply:
Good point, will do.